### PR TITLE
Docs: add infra config reference tables

### PR DESCRIPTION
**Pull request by jenmalloy**

### CODE REVIEW

File: documentation/modules/ref-brokered-infra-config-fields.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// assembly-.adoc
```
Comment (ncbaratta) [inline documentation/modules/ref-brokered-infra-config-fields.adoc]: Where is it included?

File: documentation/modules/ref-standard-infra-config-fields.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-.adoc
```
Comment (ncbaratta) [inline documentation/modules/ref-standard-infra-config-fields.adoc]: Which assembly?

File: documentation/modules/ref-brokered-infra-config-fields.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+.Brokered infrastructure configuration fields table
+[cols="40%a,60%a,options="header"]
+|===
+|Field |Description
+|version |Specifies the {ProductName} version used. When upgrading, {ProductName} uses this field to determine whether to upgrade the infrastructure to the requested version.
+|admin.resources.memory |Specifies the amount of memory allocated to the admin pod.
+|admin.podTemplate.metadata.labels |Specifies the labels added to the admin pod.
+
+ifeval::["{cmdcli}" == "kubectl"]
```
Comment (jenmalloy) [inline documentation/modules/ref-brokered-infra-config-fields.adoc]: @lulf Is there a more elegant way to handle this? If not, I'll fix the other table the same way.

Comment (lulf) [reply documentation/modules/ref-brokered-infra-config-fields.adoc]: Yeah, replace Kubernetes with the {KubePlatform} attribute

Comment (jenmalloy) [reply documentation/modules/ref-brokered-infra-config-fields.adoc]: @lulf Right! Thank you :)

**Comment by jenmalloy** [comment]
@lulf, Can you please review again with that change? Thanks!

**Comment by jenmalloy** [comment]
@ncbaratta, Can you re-review please?

**Pull request merged**